teacher.rb

```diff
@@ -0,0 +1,12 @@
+require 'person'
+
+class Teacher < Person
+  def initialize(specialization)
```

Comment:
- [ ] the initialize method should also take the arguments `age, name = 'Unknown', parent_permission: true` like in the student class

Reply:
Done